examples/index.html

```diff
@@ -22,7 +22,9 @@ <h1>Demos</h1>
 					<li><a href="../elements/pfe-icon/demo">pfe-icon</a></li>
 					<li><a href="../elements/pfe-icon-panel/demo">pfe-icon-panel</a></li>
 					<li><a href="../elements/pfe-layouts/demo">pfe-layouts</a></li>
+					<li><a href="../elements/pfe-link-list/demo">pfe-link-list</a></li>
 					<li><a href="../elements/pfe-markdown/demo">pfe-markdown</a></li>
+					<li><a href="../elements/pfe-navigation/demo">pfe-navigation</a></li>
```

Comment:
The changes in this file look like they're holdovers from another branch.

Reply:
@mwcz Thanks for reviewing. Removed pfe-navigation and pfe-link-list from the examples index file.